### ISSUE TITLE
[Feature Request] Option to limit dictionaries

### ISSUE THREAD

**Issue by BEST8OY** [xiaoyifang/goldendict#440]
**Is your feature request related to a problem? Please describe.**
When I have to many dictionaries.

**Describe the solution you'd like**
This is the workflow in my mind

**Describe the solution you'd like**
1- I have 4 dictionaries in 2 languages (one X and three Y)
2- The option I want is that **two** of those **three** Y dictionaries only **one** be shown based on a character limit (the one with more characters or less characters)
3- So when I enable this option, the shown dictionaries are as follow
X - Y - Y ----> One of the Y dictionaries is not present because the option I described

**Comment by xiaoyifang** [comment]
too specific , I think you can create a group and put your dictionaries with the order X Y Y

**Comment by BEST8OY** [comment]
> too specific , I think you can create a group and put your dictionaries with the order X Y Y

Yeah but I have to move between them all the time and it's a hassle.
If you think it's not gonna added eventually, please close the issue.
